.github/workflows/scripts.yml

```diff
@@ -0,0 +1,11 @@
+name: Lint Scripts
+on: [pull_request]
```

Comment:
Should this be on push as well?

Reply:
Hmm, that's a good question. It should be fine on just PRs, I'd be surprised if a merge caused linting issues. WDYT?

Reply:
I just used what was in another workflow. PR should be fine.